cypress/e2e/accesibility_test.cy.js

```diff
@@ -0,0 +1,20 @@
+import 'cypress-axe';
+import pages from '../routes.json';
+
+
+const replaceParams = (route) => {
+    return route;
+};
+
+
+describe('NHS Accessibility Testing', () => {
+    pages.forEach((route) => {
+        const page = replaceParams(route);
+        
+        it(`Has no WCAG 2.1 and 2.2 violation rules in route: ${page}`, () => {
```

Comment:
@AlmubdyMutaikhan I can't find where this command is set or where the ruleset is specified. Can you provide some further information?

I was expecting to see something like https://github.com/wri/wri-odp/blob/0ef333636ae628981fed4b050fbccc44deb0a221/e2e-tests/cypress/support/e2e.js#L596 on this PR.

Isn't it throwing an exception when you run the tests since the `checkAccessibility` command is not defined?

Reply:
Yes, the support folder has not been added. I guess it was omitted during transferring files into this repo.

Reply:
@AlmubdyMutaikhan can you push this folder, please?